packages/wallets/src/components/Base/Divider/Divider.tsx

```diff
@@ -8,7 +8,11 @@ type TProps = {
 };
 
 const Divider: React.FC<TProps> = ({ color = '#f2f3f4', margin, variant = 'horizontal' }) => (
-    <div className={`wallets-divider wallets-divider--${variant}`} style={{ borderColor: color, margin }} />
+    <div
+        className={`wallets-divider wallets-divider--${variant}`}
+        data-testid='dt-divider'
```

Comment:
Just a small suggestions to keep the `testid` consistent.
```suggestion
        data-testid='dt_divider'
```